src/transformers/models/deformable_detr/modeling_deformable_detr.py

```diff
@@ -315,6 +249,40 @@ def inverse_sigmoid(x, eps=1e-5):
     return torch.log(x1 / x2)
 
 
+def multi_scale_deformable_attention(
```

Comment:
Hi @alaradirik 

Saw this PR linked with Mask2Former PR so glanced through it. Noticed a minor thing so just thought of mentioning in case it's helpful.
I see that this function is added but not used in line 636. I think this file already contains another equivalent of this function called "ms_deform_attn_core_pytorch". Although the variable names used in "ms_deform_attn_core_pytorch" are a bit unclear. Would make sense to remove one of the functions to avoid duplicate code within same file.

Reply:
Nice catch, and yes, you're right. I'm removing this function